config/schedule.rb

```diff
@@ -0,0 +1,6 @@
+# Use this file to easily define all of your cron jobs.
+#
+set :output, "/tmp/cron_log.log"
```

Comment:
Just noticed, I thought I took out the set output, I probably shouldn't write to /tmp, there is another ticket to in the queue for handling writing logs to splunk, but is still open.   I'll look at what Hoan has done interim though.